code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm

```diff
@@ -96,6 +96,22 @@
 
 		occupant.visible_message("<span class='danger'>[occupant] crashed into \the [A]!</span>")
 
+/obj/structure/chair/wheelchair/plasteel
+	name = "hardened wheelchair"
+	desc = "Made from a mixture of metal-plasma sheets, this wheelchair is 3 times stronger than classic model, and is very resistant to acid and corrosion. Thanks to this it can be used in various dangerous environments without the need to worry about safety."
+	icon_state = "h_wheelchair"
+	max_integrity = 750
+	resistance_flags = ACID_PROOF
+	buildstacktype = /obj/item/stack/sheet/plasteel
+
+/obj/structure/chair/wheelchair/plastitanium
+	name = "reinforced wheelchair"
+	desc = "Made from a mixture of titanium-plasma sheets, this wheelchair is 6 times stronger than classic model, and is very resistant to acid, corrosion and fire! Thanks to this, it can be used in various dangerous environments, without the need to worry about safety... <i>but remember to not try to bathe in lava.</i>"
```

Comment:
```suggestion
	desc = "Made from a mixture of titanium-plasma sheets, this wheelchair is 6 times stronger than the classic model, and is very resistant to acid, corrosion and fire! Thanks to this it can be used in hazardous environments without much worry... <i>but remember not to try to bathe in lava.</i>"
```